ci/mypy-stubtest-allowlist.txt

```diff
@@ -32,22 +32,127 @@ matplotlib.ticker.LogitLocator.nonsingular
 matplotlib.backend_bases._Mode.__new__
 matplotlib.units.Number.__hash__
 
+# Property read-write vs read-only weirdness, fix if possible
```

Comment:
The additions in this file seem unrelated.

Reply:
These look like they are an older version of this file being reintroduced